### DISCUSSION TITLE
Download time range FROM:TO

### DISCUSSION THREAD

**Discussion by Marco-9456** [JunkFood02/Seal#474] [Tutorial]
>**Warning**
This is outdated, please use latest build which has a built-in function for this Purpose

Let's say you're watching a game play Video and you find a specific part of the video very funny and you want to download it, well just paste the following commands in a custom command template and change the time to what you want. 

```
--no-mtime
--download-sections "*01:00:00-01:05:00"
-o "/Trimmed Videos/%(uploader)s/%(title)s.%(ext)s"
```

`--download-sections "*HH:MM:SS-HH:MM:SS"`

If the video is not exactly Trimmed then add the following command:
`--force-keyframes-at-cuts`

>**Note**
This will decrease the download speed

**Comment by HassanDalwi** [comment]
How can I bookmark this

**Reply by Marco-9456** [reply]
> How can I bookmark this

Wdym?
Also this feature is already available in the app now, just download latest beta or wait for the stable release(:

**Reply by HassanDalwi** [reply]
I mean on th github

**Reply by HassanDalwi** [reply]
Do we change the time on the command itself or the command will ask us to choose the time that we prefer?

**Reply by Marco-9456** [reply]
> Do we change the time on the command itself or the command will ask us to choose the time that we prefer?

There's no need to use this anymore just download latest github release and enable format selection there you will see a Scissor icon clikc on it.

**Reply by Marco-9456** [reply]
https://user-images.githubusercontent.com/41828058/219880921-84d33853-75bc-4103-9834-a0aebc985c8b.mp4

#

**Reply by HassanDalwi** [reply]
1.8.2 ?

**Reply by Marco-9456** [reply]
> 1.8.2 ?

No, v1.9.0-beta.1

**Reply by just-max** [reply]
Was this removed again? Can't find it in 1.13.1

**Reply by Marco-9456** [reply]
> Was this removed again? Can't find it in 1.13.1

No, you just have to enable it by going to: `Settings` --> `Format` --> enable `Clip Video`

Btw, Hübsche Website :)

**Reply by just-max** [reply]
hehe, vielen Dank :)
